rxandroid/src/main/java/rx/android/schedulers/AndroidSchedulers.java

```diff
@@ -24,13 +25,17 @@ private AndroidSchedulers() {
         throw new AssertionError("No instances");
     }
 
-    private static final Scheduler MAIN_THREAD_SCHEDULER =
-            new HandlerScheduler(new Handler(Looper.getMainLooper()));
+    // See https://github.com/ReactiveX/RxAndroid/issues/238
+    // https://en.wikipedia.org/wiki/Initialization-on-demand_holder_idiom
+    private static class MainThreadSchedulerHolder {
+        private static final Scheduler MAIN_THREAD_SCHEDULER =
```

Comment:
Make this package scoped to avoid synthetic accessor method? There's already enough method calls inside of `mainThread()` below, let's not add one more.